app/wol.py

```diff
@@ -125,6 +129,16 @@ def load_computers():
 
 computers = load_computers()
 
+def send_raw_wol(mac_address, interface='eth0'):
+    mac_clean = mac_address.replace(':', '').replace('-', '')
+    mac_bytes = binascii.unhexlify(mac_clean)
```

Comment:
```suggestion
    mac_bytes = bytes.fromhex(mac_clean)
```
More readable and builtin